resources/views/home.blade.php

```diff
@@ -1,23 +1,25 @@
 @extends('layouts.app')
 
 @section('content')
-<div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">{{ __('Dashboard') }}</div>
 
-                <div class="card-body">
-                    @if (session('status'))
-                        <div class="alert alert-success" role="alert">
-                            {{ session('status') }}
-                        </div>
-                    @endif
+    <div class="lg:flex p-8">
+        <div class="lg:flex-1/2">
+           @include('__dashboard') 
+        </div>
 
-                    You are logged in, {{ Auth::user()->name }}.
+        <div class="lg:flex-1 text-left ">
+            <div class="border border-black-100">
+                <div class="p-3 font-bold">
+                    Activities
+                </div>
+
+                <hr class="mx-5">
+
+                <div class="p-3">
+                    Activities Content
                 </div>
             </div>
         </div>
```

Comment:
We can create a separate file for this one and name it as `activities.blade.php`,  and we will append in this file using `@include` directive

Reply:
Done